.github/workflows/psalm.yml

```diff
@@ -30,8 +30,8 @@ jobs:
 
     strategy:
       matrix:
-        php-versions: [ '8.1', '8.2' ]
-        server-versions: [ 'dev-master', 'dev-stable28' ]
+        php-versions: [ "8.0", "8.1", "8.2" ]
+        server-versions: [ 'dev-master', 'stable28' ]
```

Comment:
This doesn't work, I think. Use either `dev-stable28` or `28.x.x`

Reply:
yeah fixed that but waiting to push all at once after the constant thing is sorted. Is it required or should I use a class (see previous review)?